examples/center.html

```diff
@@ -13,22 +13,16 @@
   <p>Use <code>Alt</code>+<code>Shift</code>+drag to rotate the map.</p>
```

Comment:
Super minor: you have used different markup elsewhere. Nothing to worry about, but maybe we want to harmonize…